java/client/src/test/java/glide/api/models/TransactionTests.java

```diff
@@ -1210,6 +1215,9 @@ InfScoreBound.NEGATIVE_INFINITY, new ScoreBoundary(3, false), new Limit(1, 2)),
         transaction.copy("key1", "key2", true);
         results.add(Pair.of(Copy, buildArgs("key1", "key2", REPLACE_REDIS_API)));
 
+        transaction.dump("key1");
+        results.add(Pair.of(Dump, buildArgs("key1")));
+
```

Comment:
Any creative method that I can test `restore` here?

Reply:
```suggestion
        transaction.restore(new bytes[0]);
        results.add(Pair.of(Restore, buildArgs(...)));
```
or even
```suggestion
        transaction.restore("TEST".getBytes());
        results.add(Pair.of(Restore, buildArgs("TEST")));
```
Keep in mind, it is an UT, no real transaction executed there.